packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -37,7 +37,11 @@ const multiSelectComboBox = css`
 
   [part='chip'] {
     flex: 0 1 auto;
-    min-width: var(--chip-min-width);
+  }
+
+  :host(:is([readonly], [disabled])) ::slotted(input) {
+    flex-grow: 0;
+    flex-basis: 0;
```

Comment:
Should the input's padding be removed as well?

Reply:
@DiegoCardoso Thanks, done.